app/models/page.rb

```diff
@@ -10,23 +11,19 @@ class Page < ActiveRecord::Base
   belongs_to :secondary_liquid_layout, class_name: 'LiquidLayout'
   belongs_to :primary_image, class_name: 'Image'
 
-  has_many :pages_tags, dependent: :destroy
   has_many :tags, through: :pages_tags
   has_many :actions
-  has_many :images, dependent: :destroy
-  has_many :links, dependent: :destroy
+  has_many :pages_tags, dependent: :destroy
+  has_many :images,     dependent: :destroy
+  has_many :links,      dependent: :destroy
 
-  validates :title, :slug, presence: true, uniqueness: true
+  validates :title, :slug,  presence: true, uniqueness: true
   validates :liquid_layout, presence: true
-  validate :primary_image_is_owned
+  validate  :primary_image_is_owned
 
-  before_validation :create_slug
   after_save :switch_plugins
 
-  # have we thought about using friendly id? probably better
-  def create_slug
-    self.slug = title.parameterize if slug.nil? and not title.nil?
-  end
+  friendly_id :title, use: [:finders, :slugged]
 
```

Comment:
Will there still be a slug stored in the `slug` column for the page? That's required for ActionKit, as far as I know.